src/dashboard/Withdraw/RequestWithdrawBtn.tsx

```
@@ -0,0 +1,316 @@
+                                    type: 'text',
+                                    placeholder: '',
+                                    disabled: true,
+                                } }
+                            />
+                        </div>
+                    </>
+                ) : (
+                    <SimpleAlert type="warning" color="orange" label="">
+                        <p
+                            className="mb-2"
+                            dangerouslySetInnerHTML={ {
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)